red-hat-data-services/odh-deployer
# Add logic to control installation  MT-SRE secrets

- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-2446

## Dependencies

This MR should be tested with the following version of olminstall: https://gitlab.cee.redhat.com/data-hub/olminstall/-/merge_requests/18

## Testing instructions:

- On PSI/OSD with Olminstall:
  - Using olminstall, trigger first a clean build with this image quay.io/lferrnan/rhods-operator-live-catalog:1.9.3-dummy-secrets make sure that the installation is successful. ✅
  - Uninstall the operator ✅
  - Using olminstall, comment the followin line “oc create -f ./deadmansnitch-secret.yaml -n redhat-ods-monitoring” make sure the installation fails ✅
  
- Using olminstall, trigger a clean build with an old image just to be sure the changes in olminstall are backward compatible. ✅
